### ISSUE TITLE
Random crash when you run HiJackThis as a Limited user

### ISSUE THREAD

**Issue by dragokas** [dragokas/hijackthis#102]
Topic: https://safezone.cc/threads/hijackthis-fork-i-voprosy-k-razrabotchikam.28770/page-10

Dump: https://cloud.mail.ru/public/d7mo/5JCjyuGNo

```
Имя сбойного приложения: HiJackThis.exe, версия: 2.9.0.25, метка времени: 0x5f26b9be
Имя сбойного модуля: USER32.DLL, версия: 10.0.19041.388, метка времени: 0xccb63ea1
Код исключения: 0xc000041d
Смещение ошибки: 0x0003508b
Идентификатор сбойного процесса: 0x1b84
Время запуска сбойного приложения: 0x01d668f54b15a5b8
Путь сбойного приложения: C:\Temp\HiJackThis.exe
Путь сбойного модуля: C:\WINDOWS\System32\USER32.DLL
Идентификатор отчета: 6cb29352-1589-4763-b6ed-fb614e3c81b3
Полное имя сбойного пакета:
Код приложения, связанного со сбойным пакетом:
Код:
Контейнер ошибки 1786617251330192261, тип 1
Имя события: APPCRASH
Отклик: Нет данных
Идентификатор CAB: 0

Сигнатура проблемы:
P1: HiJackThis.exe
P2: 2.9.0.25
P3: 5f26b9be
P4: USER32.DLL
P5: 10.0.19041.388
P6: ccb63ea1
P7: c000041d
P8: 0003508b
P9:
P10:

Вложенные файлы:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER36AA.tmp.dmp
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER3C19.tmp.WERInternalMetadata.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER3C2A.tmp.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER3C38.tmp.csv
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER3C48.tmp.txt

Эти файлы можно найти здесь:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_HiJackThis.exe_368bf7e5da685c1f4f5fcecc6713195231f8c7_3da0cff8_0876759c-47e7-4531-b1aa-418d0dc02975

Символ анализа:
Повторный поиск решения: 0
Идентификатор отчета: 6cb29352-1589-4763-b6ed-fb614e3c81b3
Состояние отчета: 268435456
Хэшированный контейнер: a5c4476f0122f14f68cb56ec971ef385
Идентификатор GUID CAB: 0
```

**Comment by dragokas** [comment]
WinDBG report:

```

Microsoft (R) Windows Debugger Version 10.0.17134.12 X86
Copyright (c) Microsoft Corporation. All rights reserved.


Loading Dump File [C:\crash\HiJackThis.exe.11812.dmp]
User Mini Dump File with Full Memory: Only application data is available


************* Path validation summary **************
Response                         Time (ms)     Location
Deferred                                       srv*c:\Symbols*https://msdl.microsoft.com/download/symbols
Symbol search path is: srv*c:\Symbols*https://msdl.microsoft.com/download/symbols
Executable search path is: 
Windows 10 Version 19041 MP (6 procs) Free x86 compatible
Product: WinNt, suite: SingleUserTS
19041.1.x86fre.vb_release.191206-1406
Machine Name:
Debug session time: Sun Aug  9 19:54:41.000 2020 (UTC + 3:00)
System Uptime: 6 days 3:24:07.377
Process Uptime: 0 days 0:00:09.000
........................................
Loading unloaded module list
...
This dump file has an exception of interest stored in it.
The stored exception information can be accessed via .ecxr.
(2e24.308): Unknown exception - code c000041d (first/second chance not available)
eax=000a307c ebx=0000000d ecx=00000000 edx=00040a3e esi=c0000008 edi=40000000
eip=753e43ce esp=000a2f70 ebp=000a3028 iopl=0         nv up ei pl nz na po nc
cs=0023  ss=002b  ds=002b  es=002b  fs=0053  gs=002b             efl=00010202
user32!UserCallWinProcCheckWow+0x1e:
753e43ce 53              push    ebx
0:000> !analyze -v
ERROR: FindPlugIns 8007007b
ERROR: Some plugins may not be available [8007007b]
*******************************************************************************
*                                                                             *
*                        Exception Analysis                                   *
*                                                                             *
*******************************************************************************

*** ERROR: Symbol file could not be found.  Defaulted to export symbols for msvbvm60.dll - 
GetUrlPageData2 (WinHttp) failed: 12002.

KEY_VALUES_STRING: 1


TIMELINE_ANALYSIS: 1

Timeline: !analyze.Start
    Name: <blank>
    Time: 2020-08-10T09:57:34.162Z
    Diff: 61373162 mSec

Timeline: Dump.Current
    Name: <blank>
    Time: 2020-08-09T16:54:41.0Z
    Diff: 0 mSec

Timeline: Process.Start
    Name: <blank>
    Time: 2020-08-09T16:54:32.0Z
    Diff: 9000 mSec

Timeline: OS.Boot
    Name: <blank>
    Time: 2020-08-03T13:30:34.0Z
    Diff: 530647000 mSec


DUMP_CLASS: 2

DUMP_QUALIFIER: 400

CONTEXT:  (.ecxr)
eax=000a307c ebx=0000000d ecx=00000000 edx=00040a3e esi=c0000008 edi=40000000
eip=753e43ce esp=000a2f70 ebp=000a3028 iopl=0         nv up ei pl nz na po nc
cs=0023  ss=002b  ds=002b  es=002b  fs=0053  gs=002b             efl=00010202
user32!UserCallWinProcCheckWow+0x1e:
753e43ce 53              push    ebx
Resetting default scope

FAULTING_IP: 
user32!UserCallWinProcCheckWow+1e
753e43ce 53              push    ebx

EXCEPTION_RECORD:  (.exr -1)
ExceptionAddress: 753e43ce (user32!UserCallWinProcCheckWow+0x0000001e)
   ExceptionCode: c000041d
  ExceptionFlags: 00000001
NumberParameters: 0

PROCESS_NAME:  HiJackThis.exe

ERROR_CODE: (NTSTATUS) 0xc000041d - <Unable to get error code text>

EXCEPTION_CODE: (NTSTATUS) 0xc000041d - <Unable to get error code text>

EXCEPTION_CODE_STR:  c000041d

WATSON_BKT_PROCSTAMP:  5f2e8f4b

WATSON_BKT_PROCVER:  2.9.0.26

PROCESS_VER_PRODUCT:  HiJackThis Fork

WATSON_BKT_MODULE:  user32.dll

WATSON_BKT_MODSTAMP:  ccb63ea1

WATSON_BKT_MODOFFSET:  343ce

WATSON_BKT_MODVER:  10.0.19041.388

MODULE_VER_PRODUCT:  Microsoft® Windows® Operating System

BUILD_VERSION_STRING:  19041.1.x86fre.vb_release.191206-1406

MODLIST_WITH_TSCHKSUM_HASH:  60750c56989093dc6b2af5c60e99ffbe51350cdc

MODLIST_SHA1_HASH:  0adc00d2110581822df1b9f7d4fb1b73a00319e2

NTGLOBALFLAG:  0

PROCESS_BAM_CURRENT_THROTTLED: 0

PROCESS_BAM_PREVIOUS_THROTTLED: 0

APPLICATION_VERIFIER_FLAGS:  0

PRODUCT_TYPE:  1

SUITE_MASK:  272

DUMP_FLAGS:  8000c07

DUMP_TYPE:  3

ANALYSIS_SESSION_HOST:  ALEX-PC

ANALYSIS_SESSION_TIME:  08-10-2020 12:57:34.0162

ANALYSIS_VERSION: 10.0.17134.12 x86fre

THREAD_ATTRIBUTES: 
OS_LOCALE:  RUS

PROBLEM_CLASSES: 

    ID:     [0n317]
    Type:   [@APPLICATION_FAULT_STRING]
    Class:  Primary
    Scope:  DEFAULT_BUCKET_ID (Failure Bucket ID prefix)
            BUCKET_ID
    Name:   Omit
    Data:   Add
            String: [FATAL_USER_CALLBACK_EXCEPTION]
    PID:    [Unspecified]
    TID:    [Unspecified]
    Frame:  [0]

BUGCHECK_STR:  FATAL_USER_CALLBACK_EXCEPTION

DEFAULT_BUCKET_ID:  FATAL_USER_CALLBACK_EXCEPTION

PRIMARY_PROBLEM_CLASS:  FATAL_USER_CALLBACK_EXCEPTION

LAST_CONTROL_TRANSFER:  from 753e42ea to 753e43ce

STACK_TEXT:  
000a3028 753e42ea 726c2db0 00000000 0000000d user32!UserCallWinProcCheckWow+0x1e
000a308c 753d3ac8 014bb900 00000000 0000000d user32!DispatchClientMessage+0xea
000a40d0 7745357d 000a40ec 00000038 000a43cc user32!__fnOUTSTRING+0x68
000a4120 7606107c 753e3e46 00040a3e 0000000d ntdll!KiUserCallbackDispatcher+0x4d
000a4124 753e3e46 00040a3e 0000000d 00000001 win32u!NtUserMessageCall+0xc
000a415c 753d115b 0000000d 00000001 02cb1040 user32!CallWindowProcAorW+0xfe
000a4174 6605d082 ffff053d 00040a3e 0000000d user32!CallWindowProcA+0x1b
WARNING: Stack unwind information not available. Following frames may be wrong.
000a41e0 660ac5da 032b34c8 00040a3e 0000000d msvbvm60!IID_IVbaHost+0x2ea42
000a4248 6605ce49 02ca101c 00040a3e 0000000d msvbvm60!DllCanUnloadNow+0xc928
000a4270 6605f97d 02ca101c 00040a3e 0000000d msvbvm60!IID_IVbaHost+0x2e809
000a42cc 753ed9cb 00040a3e 0000000d 00000001 msvbvm60!IID_IVbaHost+0x3133d
000a42f8 753e46ea 6605f74e 00040a3e 0000000d user32!_InternalCallWinProc+0x2b
000a43dc 753e3a5d 6605f74e 00000000 0000000d user32!UserCallWinProcCheckWow+0x33a
000a4440 753d29aa 014bb900 00000000 02cb1040 user32!SendMessageWorker+0x1fd
000a4488 660ac746 00040a3e 02cb1040 00000001 user32!GetWindowTextA+0x5a
000a44e8 6605ce49 02ca101c 00040a3e 0000000c msvbvm60!DllCanUnloadNow+0xca94
000a4510 6605f97d 02ca101c 00040a3e 0000000c msvbvm60!IID_IVbaHost+0x2e809
000a456c 753ed9cb 00040a3e 0000000c 00000000 msvbvm60!IID_IVbaHost+0x3133d
000a4598 753e46ea 6605f74e 00040a3e 0000000c user32!_InternalCallWinProc+0x2b
000a467c 753e3a5d 6605f74e 00000000 0000000c user32!UserCallWinProcCheckWow+0x33a
000a46e4 753d4069 014bb900 00000000 055f2524 user32!SendMessageWorker+0x1fd
000a4704 660abd77 00040a3e 055f2524 00000001 user32!SetWindowTextA+0x39
000a472c 6606cdb0 00000000 055f2524 055f2524 msvbvm60!DllCanUnloadNow+0xc0c5
000a4768 6606d0a1 02ca101c 0000000b 055f2524 msvbvm60!IID_IVbaHost+0x3e770
000a479c 6603bcc6 032b06cc 0000000b 00000004 msvbvm60!IID_IVbaHost+0x3ea61
000a49b8 0081f44b 00c134d0 660da274 ffffffff msvbvm60!IID_IVbaHost+0xd686
000a4bbc 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a4dc0 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a4fc4 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a51c8 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a53cc 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a55d0 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a57d4 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a59d8 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a5bdc 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a5de0 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a5fe4 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a61e8 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a63ec 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a65f0 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a67f4 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a69f8 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a6bfc 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a6e00 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a7004 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a7208 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a740c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a7610 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a7814 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a7a18 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a7c1c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a7e20 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a8024 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a8228 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a842c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a8630 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a8834 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a8a38 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a8c3c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a8e40 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a9044 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a9248 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a944c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a9650 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a9854 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a9a58 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a9c5c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000a9e60 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aa064 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aa268 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aa46c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aa670 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aa874 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aaa78 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aac7c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aae80 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ab084 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ab288 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ab48c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ab690 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ab894 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aba98 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000abc9c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000abea0 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ac0a4 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ac2a8 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ac4ac 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ac6b0 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ac8b4 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000acab8 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000accbc 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000acec0 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ad0c4 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ad2c8 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ad4cc 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ad6d0 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ad8d4 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000adad8 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000adcdc 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000adee0 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ae0e4 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ae2e8 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ae4ec 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ae6f0 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000ae8f4 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aeaf8 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aecfc 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aef00 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000af104 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000af308 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000af50c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000af710 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000af914 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000afb18 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000afd1c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000aff20 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b0124 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b0328 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b052c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b0730 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b0934 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b0b38 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b0d3c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b0f40 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b1144 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b1348 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b154c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b1750 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b1954 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b1b58 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b1d5c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b1f60 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b2164 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b2368 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b256c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b2770 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b2974 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b2b78 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b2d7c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b2f80 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b3184 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b3388 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b358c 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b3790 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b3994 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b
000b3b98 0081f44b 00c134d0 660da274 ffffffff HiJackThis!frmMain::LoadSettings+0x3d8b

THREAD_SHA1_HASH_MOD_FUNC:  f4bed189523612f59e38a44e575da3cf5f7af2fb

THREAD_SHA1_HASH_MOD_FUNC_OFFSET:  79d89a401ed7ee3ca58d21abddd08ecaa16ceef2

THREAD_SHA1_HASH_MOD:  104edd22f60b92d91c1e9aa9b6e32cf63d938fd6

FOLLOWUP_IP: 
win32u!NtUserMessageCall+c
7606107c c21c00          ret     1Ch

FAULT_INSTR_CODE:  90001cc2

SYMBOL_STACK_INDEX:  4

SYMBOL_NAME:  win32u!NtUserMessageCall+c

FOLLOWUP_NAME:  MachineOwner

MODULE_NAME: win32u

IMAGE_NAME:  win32u.dll

DEBUG_FLR_IMAGE_TIMESTAMP:  55cf9768

STACK_COMMAND:  ~0s ; .ecxr ; kb

BUCKET_ID:  FATAL_USER_CALLBACK_EXCEPTION_win32u!NtUserMessageCall+c

FAILURE_EXCEPTION_CODE:  c000041d

FAILURE_IMAGE_NAME:  win32u.dll

BUCKET_ID_IMAGE_STR:  win32u.dll

FAILURE_MODULE_NAME:  win32u

BUCKET_ID_MODULE_STR:  win32u

FAILURE_FUNCTION_NAME:  NtUserMessageCall

BUCKET_ID_FUNCTION_STR:  NtUserMessageCall

BUCKET_ID_OFFSET:  c

BUCKET_ID_MODTIMEDATESTAMP:  55cf9768

BUCKET_ID_MODCHECKSUM:  179ad

BUCKET_ID_MODVER_STR:  10.0.19041.388

BUCKET_ID_PREFIX_STR:  FATAL_USER_CALLBACK_EXCEPTION_

FAILURE_PROBLEM_CLASS:  FATAL_USER_CALLBACK_EXCEPTION

FAILURE_SYMBOL_NAME:  win32u.dll!NtUserMessageCall

FAILURE_BUCKET_ID:  FATAL_USER_CALLBACK_EXCEPTION_c000041d_win32u.dll!NtUserMessageCall

WATSON_STAGEONE_URL:  http://watson.microsoft.com/StageOne/HiJackThis.exe/2.9.0.26/5f2e8f4b/user32.dll/10.0.19041.388/ccb63ea1/c000041d/000343ce.htm?Retriage=1

TARGET_TIME:  2020-08-09T16:54:41.000Z

OSBUILD:  19041

OSSERVICEPACK:  292

SERVICEPACK_NUMBER: 0

OS_REVISION: 0

OSPLATFORM_TYPE:  x86

OSNAME:  Windows 10

OSEDITION:  Windows 10 WinNt SingleUserTS

USER_LCID:  0

OSBUILD_TIMESTAMP:  unknown_date

BUILDDATESTAMP_STR:  191206-1406

BUILDLAB_STR:  vb_release

BUILDOSVER_STR:  10.0.19041.1.x86fre.vb_release.191206-1406

ANALYSIS_SESSION_ELAPSED_TIME:  18824

ANALYSIS_SOURCE:  UM

FAILURE_ID_HASH_STRING:  um:fatal_user_callback_exception_c000041d_win32u.dll!ntusermessagecall

FAILURE_ID_HASH:  {97c1631f-57be-0eb2-ff73-82a0f380558f}

Followup:     MachineOwner
---------

```

**Comment by dragokas** [comment]
Temporary fix for this issue.
[fix.zip](https://github.com/dragokas/hijackthis/files/5057669/fix.zip)

**Comment by dragokas** [comment]
Fixed in v2.9.0.27
https://github.com/dragokas/hijackthis/commit/4bc9785d420119bb0a25cc60ddaa64b8b0ca5ec2
